packages/ppx/src/declarations_to_emotion.re

```diff
@@ -305,6 +305,10 @@ let render_length = (~loc) =>
   | `Zero => [%expr `zero];
 
 let rec render_function_calc = (~loc, calc_sum) => {
+  render_calc_sum(~loc, calc_sum);
+}
+
+and render_calc_sum = (~loc, calc_sum) => {
```

Comment:
I re-assigned the function so that it is intuitive when we call `render_function_calc` than `render_calc_sum`. For example `...Function_calc(fc) => render_function_calc(~loc, fc)`